opsdroid/testing/fixtures.py

```diff
@@ -15,3 +18,11 @@ def opsdroid():
     """
     with OpsDroid(config={}) as opsdroid:
         yield opsdroid
+
+
+@pytest.fixture
+def _tmp_dir():
+    _tmp_dir = os.path.join(tempfile.gettempdir(), "opsdroid_tests")
+    with contextlib.suppress(FileExistsError):
+        os.makedirs(_tmp_dir, mode=0o777)
+    return _tmp_dir
```

Comment:
This isn't quite how fixtures work and there are also cleaner ways to do this.

I don't think we should be using a fixture for this. See my other comment.

Reply:
true